### DISCUSSION TITLE
Is pinging redundant?

### DISCUSSION THREAD

**Discussion by adamleantech** [redisson/redisson#5290] [Q&A]
the [configuration](https://github.com/redisson/redisson/wiki/2.-Configuration/#pingconnectioninterval-6) mentions a netty issue which I cannot find a reference for.  Is there any purpose for this now?

**Comment by mrniko** [comment]
It's not an issue. If you connected to Redis through proxy or any other service then only ping can detect dead connections.

**Reply by adamleantech** [reply]
@mrniko many thanks for the swift response, this is very helpful.  Assuming the scenario where a proxy is being used and a dead connection occurs without being detected by the redisson client, I assume a command issued on that connection would fail and be retried?
What I'm trying to establish is how essential pinging is and whether disabling it would have significant negative consequences.  At the moment we see hundreds of pings per minute from each client which results in many thousand across all our microservice instances which in itself isn't an issue but it creates noise when trying to drill down on other issues
